templates/inspec/integration_test_template.erb

```diff
@@ -1,4 +1,9 @@
 <%= lines(autogen_notice :ruby) -%>
+<% vcr = ENV['VCR_MODE'] -%>
```

Comment:
I didn't realize that this ENV variable has expected values. Could you do some kind of check to throw an error if this var has a bad value?